packages/cf-component-input/src/Input.js

```diff
@@ -19,14 +19,7 @@ class Input extends React.Component {
     }
 
     return (
-      <input
-        type={this.props.type}
-        className={className}
-        name={this.props.name}
-        value={this.props.value}
-        onChange={this.handleChange}
-        placeholder={this.props.placeholder}
-        autoComplete={this.props.autoComplete}/>
+      <input className={className} onChange={this.handleChange} {...this.props} />
```

Comment:
Don't think this will use `this.handleChange` as expected

>The specification order is important. Later attributes override previous ones.
https://gist.github.com/sebmarkbage/07bbe37bc42b6d4aef81#spread-attributes